Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ExternalCalls.aspx.cs

```diff
@@ -20,9 +20,9 @@ public partial class ExternalCalls : System.Web.UI.Page
         private const string ConnectionString = @"Data Source=.\SQLEXPRESS;Initial Catalog=RDDTestDatabase;Integrated Security=True";
 
         /// <summary>
-        /// Valid SQL Query.
+        /// Valid SQL Query. The wait for delay of 6msec is used to prevent access time of less than 1msec. SQL is not accurate below 3, so used 6 msec delay.
         /// </summary> 
-        private const string ValidSqlQueryToApmDatabase = "WAITFOR DELAY '00:00:00:001'; select * from dbo.Messages";
+        private const string ValidSqlQueryToApmDatabase = "WAITFOR DELAY '00:00:00:006'; select * from dbo.Messages";
```

Comment:
This is what I'm talking about when I say it is a race condition.
You are chasing a timing value that "happens to work" currently and this is the third time changing it.
What about changing the criteria to not fail super low values?
Why is "zero" a problem?